internal/pkg/chain/sampler.go

```diff
@@ -43,19 +55,11 @@ func (s *Sampler) Sample(ctx context.Context, head block.TipSetKey, epoch abi.Ch
 			return nil, err
 		}
 	} else {
-		// Sampling for the genesis block.
-		ticket.VRFProof = []byte{}
-	}
-
-	buf := bytes.Buffer{}
-	buf.Write(ticket.VRFProof)
-	err := binary.Write(&buf, binary.BigEndian, epoch)
-	if err != nil {
-		return nil, err
+		// Sampling for the genesis state or genesis tipset.
+		ticket = s.genesisTicket
```

Comment:
FYI @sternhenri a genesis ticket is now used instead of empty, and that ticket still mixed with the epoch number. The same ticket is used whenever there is no parent block: both pre-genesis state and the evaluation of the genesis block.